board/views.py

```diff
@@ -1,13 +1,111 @@
 import os
 import datetime
-import requests
+from requests import post, get
 from django.shortcuts import render
 from django.core.paginator import Paginator, EmptyPage, PageNotAnInteger
 
 
-IGDB_URL = 'https://api.igdb.com/v4/'
-IGDB_HEADERS = {'Client-ID': os.getenv('API_CLIENT_ID'),
-                'Authorization': os.getenv('API_SECRET_KEY')}
+API_IGDB_URL = 'https://api.igdb.com/v4/'
+API_TWITTER_URL = 'https://api.twitter.com/2/'
+
+
+class Twitter:
+    def __init__(self, bearer_token):
+        self.bearer_token = bearer_token
+
+    def api_request(self, endpoint, query):
+        url = f'{API_TWITTER_URL}{endpoint}'
+        params = self._compose_request(query)
+        response = get(url, **params)
+        response.raise_for_status()
+        response = response.json()
+        if 'data' in response:
+            return response['data']
+        else:
+            return ()
```

Comment:
Better to return `None` here:
```suggestion
        return response.get('data')
```